.goreleaser.yml

```diff
@@ -7,6 +7,17 @@ builds:
     ldflags:
       - -X github.com/vmware-tanzu/asset-relocation-tool-for-kubernetes/cmd.Version={{ .Version }}
     binary: relok8s
+dockers:
+  -
+    ids:
+      - release
+    dockerfile: Dockerfile
+    image_templates:
+      - "ghcr.io/vmware-tanzu/asset-relocation-tool-for-kubernetes:{{ .Tag }}"
+      - "ghcr.io/vmware-tanzu/asset-relocation-tool-for-kubernetes:latest"
```

Comment:
Does this mean the image is living here and not on `projects.registry.vmware.com/tanzu_isv_engineering/relok8s`? We should update the pipeline.yaml to find the image in the right location, then.

Reply:
yeah, sorry, I missed linking the task https://github.com/vmware-tanzu/asset-relocation-tool-for-kubernetes/issues/127

This whole task started because I noticed that we are no longer building the external container images since we've moved the release process outside.